app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -1,21 +1,50 @@
 <%
   email_signup_link ||= false
+  email_signup_link_text ||= "Get email alerts"
```

Comment:
Should this and the default feed link text be in locale files for translation?

Reply:
Not a bad idea, will add.